### PR TITLE
[PR] Update README for more clarity

### PR DESCRIPTION
**Pull request by commit-master**
Added in comment the other option using `credentials` instead of `keyFilename`. 
Also added comments to explain the difference between both.

### CODE REVIEW

**Comment by commit-master** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -78,11 +78,19 @@ If you are not running this client on Google Compute Engine, you need a Google D
 // Authenticating on a global basis.
 var projectId = process.env.GCLOUD_PROJECT; // E.g. 'grape-spaceship-123'
 
+// Using a string of your keyfile.json path
 var gcloud = require('gcloud')({
   projectId: projectId,
   keyFilename: '/path/to/keyfile.json'
 });
```
Comment (stephenplusplus) [inline README.md]: How about:

```js
var gcloud = require('gcloud')({
  projectId: projectId,
  
  // The path to your key file:
  keyFilename: '/path/to/keyfile.json',

  // Or the contents of the key file:
  credentials: require('./path/to/keyfile.json')
});
```

File: README.md [hunk 2]
```diff
@@ -78,11 +78,19 @@ If you are not running this client on Google Compute Engine, you need a Google D
 // Authenticating on a global basis.
 var projectId = process.env.GCLOUD_PROJECT; // E.g. 'grape-spaceship-123'
 
+// Using a string of your keyfile.json path
```
Comment (stephenplusplus) [inline README.md]: Lose the heading.

File: README.md [hunk 3]
```diff
@@ -78,11 +78,19 @@ If you are not running this client on Google Compute Engine, you need a Google D
...
 var gcloud = require('gcloud')({
   projectId: projectId,
   keyFilename: '/path/to/keyfile.json'
 });
 
+// Or you can use your keyfile.json directly
```
Comment (stephenplusplus) [inline README.md]: Lose this object if we go with the idea from https://github.com/GoogleCloudPlatform/gcloud-node/pull/1446/files#r71899332

File: README.md [hunk 4]
```diff
@@ -78,11 +78,19 @@ If you are not running this client on Google Compute Engine, you need a Google D
...
+// var gcloud = require('gcloud')({
+//   projectId: projectId,
+//   credentials: keyfile.json
+// });
+
+// REMINDER : keyFilename requires string of your json path meanwhile credentials requires an object.
```
Comment (stephenplusplus) [inline README.md]: I don't think this is necessary if we implement https://github.com/GoogleCloudPlatform/gcloud-node/pull/1446/files#r71899332

**Comment by stephenplusplus** [comment]
Thanks :D

**Comment by commit-master** [comment]
This is my first PR actually ahah. Sounds very good to me ! Way more clear than my attempt. 

Thanks !

**Comment by stephenplusplus** [comment]
Oh that's awesome, thanks for spending it on helping us! I remember [my first PR](https://github.com/tastejs/todomvc/pull/342)... ah, the good ol' days.

Hey, now you can change your name to `pr-master` :)

**Comment by commit-master** [comment]
Ahaha thanks ! Appreciate it !
💥 ⚡️ 🎉

**Comment by stephenplusplus** [comment]
Looks great to me. Thanks again, feel free to send us many more :)

**Comment by commit-master** [comment]
Thank you, I feel complete now 🎆

**Comment by stephenplusplus** [comment]
Hahaha 🌠 💯 🏁

**Pull request merged**